app/assets/v2/css/lib/slider.css

```diff
@@ -0,0 +1,46 @@
+.g-switch input { 
+  display:none;
```

Comment:
Why is there different formatting/style in this CSS file? Also, should we get a CSS style checker/linter going?

Reply:
You mean the spacing right? I'll fix that up and yeah we should